scanners/reactnative/plain.go

```diff
@@ -404,3 +317,22 @@ func (scanner *Scanner) defaultConfigs() (models.BitriseConfigMap, error) {
 
 	return configMap, nil
 }
+
+func (scanner *Scanner) addTestSteps(configBuilder *models.ConfigBuilderModel, workflowID models.WorkflowID, workDir string) {
```

Comment:
I think it would be better if this function simply returned the list of steps instead of directly calling the config builder through a pointer. Like `steps.DefaultDeployStepListV2()`